frontend/src/modules/entitydetails/components/EntityDetails.js

```diff
@@ -213,6 +213,8 @@ export class EntityDetailsBase extends React.Component<InternalProps, State> {
                 locale={ state.locale }
                 machinery={ state.machinery }
                 otherlocales={ state.otherlocales }
+                orderedOtherLocales={ state.orderedOtherLocales }
+                preferredCount={ state.preferredCount }
```

Comment:
In this context, it's unclear what `preferredCount` refers to. Could you rename that to `preferredLocalesCount` at least until it's in a context where we know it's about locales?

Reply:
OK, `preferredOtherLocalesCount` even.